contracts/GoodGhosting.sol

```diff
@@ -25,6 +25,10 @@ contract GoodGhosting is Ownable, Pausable {
     //  total principal amount
     uint256 public totalGamePrincipal;
 
+    uint256 public adminFee;
```

Comment:
Really minor comment - Just for readability purposes, I'd suggest we change the name of this variable to better represent it's purposes.
Seems like it contains the amount that the admin can withdraw, so it's better to call something like `adminFeeAmount` to it's a bit more clear.

Just because if we have only `adminFee`, it's easy to get confused and wonder if that's applied to the `fee percentage` or `fee amount`.